spec/cb/clients/resume_insights_spec.rb

```diff
@@ -0,0 +1,65 @@
+# Copyright 2016 CareerBuilder, LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+require 'spec_helper'
+
+module Cb
+  describe Cb::Clients::ResumeInsights do
+    let(:headers) do
+      {
+        'Accept'=>'application/json',
+        'Accept-Encoding'=>'deflate, gzip',
+        'Authorization'=>'Bearer token',
+        'Content-Type' => 'application/json',
+        'Developerkey'=> Cb.configuration.dev_key
+      }
+    end
+    let(:response_template) { { 'page' => -1, 'page_size' => -1, 'total' => 0 } }
+    let(:success_response) { { 'data' => [ data ].flatten }.merge(response_template) }
+
+    describe '#keywords' do
+      let(:uri) { "https://api.careerbuilder.com/consumer/insights/keywords/id?developerkey=#{ Cb.configuration.dev_key }&outputjson=true" }
+
+      context 'asking for a specific top 10 keyword list by id' do
+        let(:response) { JSON.parse File.read('spec/support/response_stubs/resume_insights/keywords.json') }
```

Comment:
Could we do `JSON.parse(File.read('path'))` - that's more in line what we've been doing, and is internally consistent.